Convert `.show()` to `.root_group =` for CircuitPython 9

in examples and README.rst

I'm not sure of this error from the CI:

```Warning, treated as error:
/home/runner/work/Adafruit_CircuitPython_ACeP7In/Adafruit_CircuitPython_ACeP7In/adafruit_acep7in.py:docstring of adafruit_acep7in.ACeP7In:15:'any' reference target not found: displayio.FourWire
Error: Process completed with exit code 2.
```

I can't find any other instances of `displayio.FourWire`

--- a/README.rst
+++ b/README.rst
@@ -105,6 +105,7 @@ Usage Example
     import time
     import board
     import displayio
+    import fourwire
     import adafruit_acep7in
 
     displayio.release_displays()
@@ -116,7 +117,7 @@ Usage Example
     epd_reset = board.D11
     epd_busy = board.D12
 
-    display_bus = displayio.FourWire(
+    display_bus = fourwire.FourWire(
         spi, command=epd_dc, chip_select=epd_cs, reset=epd_reset, baudrate=1000000
     )
 
@@ -133,7 +134,7 @@ Usage Example
         t = displayio.TileGrid(pic, pixel_shader=pic.pixel_shader)
         g.append(t)
 
-        display.show(g)
+        display.root_group = g
 
         display.refresh()
 

--- a/README.rst
+++ b/README.rst
@@ -105,8 +105,13 @@ Usage Example
     import time
     import board
     import displayio
-    import fourwire
     import adafruit_acep7in
+
+    # For 8.x.x and 9.x.x. When 8.x.x is discontinued as a stable release, change this.
+    try:
+        from fourwire import FourWire
+    except ImportError:
+        from displayio import FourWire
 
     displayio.release_displays()
 
@@ -117,7 +122,7 @@ Usage Example
     epd_reset = board.D11
     epd_busy = board.D12
 
-    display_bus = fourwire.FourWire(
+    display_bus = FourWire(
         spi, command=epd_dc, chip_select=epd_cs, reset=epd_reset, baudrate=1000000
     )
 

--- a/adafruit_acep7in.py
+++ b/adafruit_acep7in.py
@@ -21,7 +21,13 @@ Implementation Notes
 
 """
 
-import displayio
+# For 8.x.x and 9.x.x. When 8.x.x is discontinued as a stable release, change this.
+try:
+    from fourwire import FourWire
+    from epaperdisplay import EPaperDisplay
+except ImportError:
+    from displayio import FourWire
+    from displayio import EPaperDisplay
 
 __version__ = "0.0.0+auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ACeP7In.git"
@@ -53,7 +59,7 @@ _STOP_SEQUENCE = b"\x02\x01\x00"  # Power off only
 
 
 # pylint: disable=too-few-public-methods
-class ACeP7In(displayio.EPaperDisplay):
+class ACeP7In(EPaperDisplay):
     r"""Display driver for 7" ACeP epaper display. Driver IC name is unknown.
 
     :param bus: The data bus the display is on
@@ -69,7 +75,7 @@ class ACeP7In(displayio.EPaperDisplay):
           Display rotation
     """
 
-    def __init__(self, bus, **kwargs):
+    def __init__(self, bus: FourWire, **kwargs):
         width = kwargs["width"]
         height = kwargs["height"]
         if "rotation" in kwargs and kwargs["rotation"] % 180 != 0:

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -27,7 +27,7 @@ extensions = [
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["displayio"]
 
 autodoc_preserve_defaults = True
 

--- a/examples/acep7in_simpletest.py
+++ b/examples/acep7in_simpletest.py
@@ -11,6 +11,7 @@
 import time
 import board
 import displayio
+import fourwire
 import adafruit_acep7in
 
 displayio.release_displays()
@@ -22,7 +23,7 @@ epd_dc = board.D10
 epd_reset = board.D11
 epd_busy = board.D12
 
-display_bus = displayio.FourWire(
+display_bus = fourwire.FourWire(
     spi, command=epd_dc, chip_select=epd_cs, reset=epd_reset, baudrate=1000000
 )
 
@@ -39,7 +40,7 @@ with open(fn, "rb") as f:
     t = displayio.TileGrid(pic, pixel_shader=pic.pixel_shader)
     g.append(t)
 
-    display.show(g)
+    display.root_group = g
 
     display.refresh()
 

--- a/examples/acep7in_simpletest.py
+++ b/examples/acep7in_simpletest.py
@@ -11,8 +11,14 @@
 import time
 import board
 import displayio
-import fourwire
 import adafruit_acep7in
+
+# For 8.x.x and 9.x.x. When 8.x.x is discontinued as a stable release, change this.
+try:
+    from fourwire import FourWire
+except ImportError:
+    from displayio import FourWire
+
 
 displayio.release_displays()
 
@@ -23,7 +29,7 @@ epd_dc = board.D10
 epd_reset = board.D11
 epd_busy = board.D12
 
-display_bus = fourwire.FourWire(
+display_bus = FourWire(
     spi, command=epd_dc, chip_select=epd_cs, reset=epd_reset, baudrate=1000000
 )
 

